cmd/transactioncmd/transaction_sign.go

```diff
@@ -112,7 +123,7 @@ func signTx(_ *cobra.Command, args []string) error {
 			}
 			subnetID = sc.Networks[network.Name()].SubnetID
 			if subnetID == ids.Empty {
```

Comment:
are we currently displaying what tx id is going to be signed before user is signing it?

Reply:
something like u will be signing tx.... and then prompt if user wants to continue

Reply:
seems like ur already doing it for convert, i guess wondering for createchain / createsubnet but seems that this will be less common now, feel free to ignore

Reply:
tx id depends on signature

Reply:
we can add for other txs types later on, this is for convert only